jsapp/js/editorMixins/editableForm.js

```diff
@@ -693,32 +696,57 @@ export default Object.assign(
     },
 
     renderBackgroundAudioWarning() {
-      return (
-        <bem.FormBuilderMessageBox m='warning'>
-          <span data-tip={t('background recording')}>
-            <i className='k-icon k-icon-project-overview' />
-          </span>
+      if (this.state.isBackgroundAudioBannerDismissed) return null
+      let bannerText = t(
+        'This form will automatically [record audio in the background](##SUPPORT_LINK##). Consider adding an acknowledgement note to inform respondents or data collectors that they will be recorded while completing this survey.',
+      )
 
-          <p>
-            {t(
-              'This form will automatically record audio in the background. Consider adding an acknowledgement note to inform respondents or data collectors that they will be recorded while completing this survey. This feature is available in ',
-            )}
-            <a
-              title='Install KoBoCollect'
-              target='_blank'
-              href='https://play.google.com/store/apps/details?id=org.koboc.collect.android'
-            >
-              {t('Collect version 1.30 and above')}
-            </a>
-            {'.'}
-          </p>
-
-          {envStore.isReady && envStore.data.support_url && (
-            <a href={envStore.data.support_url + RECORDING_SUPPORT_URL} target='_blank' data-tip={t('help')}>
-              <Icon name='help' size='s' />
-            </a>
-          )}
-        </bem.FormBuilderMessageBox>
+      let supportLink = ''
+      if (envStore.isReady && envStore.data.support_url) {
+        supportLink = envStore.data.support_url + RECORDING_SUPPORT_URL
+      }
+      bannerText = bannerText.replace('##SUPPORT_LINK##', supportLink)
+
+      return (
+        <Group pos='relative' wrap='nowrap' p='xs' bd='1px solid blue.7' bg='blue.9'>
```

Comment:
I think we need some padding on the last two elements of this group:
<img width="1504" height="350" alt="image" src="https://github.com/user-attachments/assets/247eca88-ac5a-4992-aca4-ad9fce5574aa" />